lms/djangoapps/api_manager/courses/views.py

```diff
@@ -38,10 +38,15 @@
 from xmodule.modulestore.django import modulestore
 from xmodule.modulestore.search import path_to_location
 
-from api_manager.courseware_access import get_course, get_course_child, get_course_leaf_nodes, get_course_key, \
+from api_manager.courseware_access import get_course, get_course_child, get_course_key, \
     course_exists, get_modulestore, get_course_descriptor
-from api_manager.models import CourseGroupRelationship, CourseContentGroupRelationship, GroupProfile
+from api_manager.models import (
+    CourseGroupRelationship,
+    CourseContentGroupRelationship,
+    GroupProfile,
+)
 from progress.models import CourseModuleCompletion
+from openedx.core.djangoapps.content.course_metadata.models import CourseAggregatedMetaData
```

Comment:
This line (as well as the progress import) should be relocated to above the api_manager imports